### DISCUSSION TITLE
如何在不设置动画的情况下给pagview设置一个图片占位

### DISCUSSION THREAD

**Discussion by RainlyTree** [Tencent/libpag#1709] [Q&A] [answered]
场景：pag动画需要下载 如何在还没下载好的情况下 先设置一张图片显示 等到下载好pag动画后 我再去加载动画呢

**Comment by kevingpqi123** [comment]
PAGView 本质上就是一个 View，Android 平台是一个 TextureView，iOS 平台是 UIView，你看下平台层是否有相关设置方法

**Comment by domchen** [comment, marked as answer]
这个需要业务自己封装，你可以弄个容器里面做两层，背后放个图片前面放个PAGView，暴露一个接口设置背景图。自己扩展这种特殊的业务需求变成项目里的组件就行。
